README.md

```diff
@@ -197,6 +197,11 @@ they exist in the local directory:
   must have an access mode of 0600 or less. If this file does not
   exist and encryption is requested the user is queried instead.
 
+* `mkosi.secure-boot.crt` and `mkosi.secure-boot.key` may contain a
```

Comment:
I think this should be “a**n** X509 certificate” (pronunciation of X509 starts with vowel).